ansible/roles/contiv_network/tasks/main.yml

```diff
@@ -21,33 +33,12 @@
   copy: src=netmaster.service dest=/etc/systemd/system/netmaster.service
   when: run_as == "master"
 
+# resolves an issue with netmaster: https://github.com/contiv/netplugin/issues/210
 - name: start netmaster
   shell: systemctl daemon-reload && systemctl start netmaster
   when: run_as == "master"
+  run_once: true
 
-- name: copy environment file for netplugin
-  copy: src=netplugin dest=/etc/default/netplugin
-
-- name: copy systemd units for netplugin
-  copy: src=netplugin.service dest=/etc/systemd/system/netplugin.service
-
-- name: start netplugin
-  shell: systemctl daemon-reload && systemctl start netplugin
-
-- name: setup netmaster host alias
-  shell: echo "{{ service_vip }} netmaster" >> /etc/hosts
-
-# XXX: need to move the following to correct roles
+# FIXME: need to move the following to correct roles -- also needs release tooling
```

Comment:
Why add FIXME? Can we not fix it? What is the reason to take it in a separate PR?